addon/components/frost-button.js

```diff
@@ -222,5 +223,12 @@ export default Ember.Component.extend({
     if (!this.get('disabled') && _.isFunction(this.attrs['onClick'])) {
       this.attrs['onClick'](this.get('id'))
     }
+  }),
+
+  _onFocus: Ember.on('focusIn', function (e) {
+    // If an onFocus handler is defined, call it
+    if (this.attrs.onFocus) {
+      this.attrs.onFocus()
+    }
```

Comment:
I find defining arbitrary keys like this odd when they are not used and personally think it would be better to define this in the `init` method. Also it would probably make sense to pass the argument to the consmuer `onFocus` property as well.
```javascript
init () {
  this._super(...arguments)

  Ember.on('focusIn', (e) => {
    const onFocus = this.get('onFocus')

    if (onFocus) {
      onFocus(e)
    }
  })
}
```